analyzers/its/expected/AnalyzeGenerated.VB/AnalyzeGeneratedVb-{C4F122A1-A51D-4194-998F-686A6EF31E44}-S2259.json

```diff
@@ -0,0 +1,95 @@
+{
+"issues":  [
+{
+"id":  "S2259",
+"message":  "'obj' is Nothing on at least one execution path.",
+"location":  {
+"uri":  "sources\GeneratedCode.VB\CompilerGenerated.vb",
```

Comment:
This file is coming from `AnalyzeGenerated.VB` project that has a setting to analyze also generated file.
It works as expected, as there's not counter partfrom `SkipGenerated.VB` project.

Both projects share the same source files from `GeneratedCode.VB` directory.